docs/admin/dbedit.md

```diff
@@ -0,0 +1,287 @@
+```
+
+dbfile and jsonfile flags are mandatory and the path for db file should not exist.
+
+Examples
+
+```
+# heketi db import --jsonfile=heketi-db.json --dbfile=heketi-new.db  
+DB imported to heketi-new.db
+```
+
+This will create a db file with contents that reflect the contents of the json file.
+
+## delete-bricks-with-empty-path command
+delete-bricks-with-empty-path command removes brick entries from db that have empty path. It also adds back free space into the device from which the brick is removed. In earlier versions of heketi if heketi process was killed during creation of a volume, it would have left bricks that were created but not part of any volume in heketi db. To clean such bricks you may use this command. To identify such bricks use the following command:
+
+```
+# heketi-cli topology info | grep ".*Path:[\ \]$"
+                                Id:8ab5071a6acbfd905c00ff9ee627d940   Size (GiB):5       Path: 
+```
+
+```
+# heketi db delete-bricks-with-empty-path [flags]
+```
+
+### flags
+
+
+```
+    --all                    if set true, then all bricks with empty path are removed
+    --clusters stringSlice   comma separated list of cluster IDs
+    --dbfile string          File path for db to operate on
+    --debug                  Show debug logs on stdout
+    --devices stringSlice    comma separated list of device IDs
+-h, --help                   help for delete-bricks-with-empty-path
+    --nodes stringSlice      comma separated list of node IDs
+```
+
+
+Examples
+
+```
+# heketi db delete-bricks-with-empty-path --dbfile ./heketi.db 
+neither --all flag nor list of clusters/nodes/devices is given
+
+# heketi db delete-bricks-with-empty-path --dbfile ./heketi.db --all
+bricks with empty path removed
+
+# heketi db delete-bricks-with-empty-path --dbfile ./heketi.db --clusters 36ac90baf26e1965feb15fe1d6e302c6
+bricks with empty path removed 
+
+# heketi db delete-bricks-with-empty-path --dbfile ./heketi.db --nodes 98ac90baf26e1965feb15fe1d6e302e3,acac90baf26e1965feb15fe1d6e302fd
+bricks with empty path removed 
+
+# heketi db delete-bricks-with-empty-path --dbfile ./heketi.db --devices 98ac90baf26e1965feb15fe1d6e302e3,acac90baf26e1965feb15fe1d6e302fd --clusters 36ac90baf26e1965feb15fe1d6e302c6
+bricks with empty path removed 
+```
+
+
+## delete-pending-entries
+delete-pending-entries removes entries from db that have pending attribute set. Heketi version 6.0 onward, if heketi process is terminated while an operation is being performed incomplete operations would be saved in pending operations bucket. When restarted, heketi process uses this information to differentiate completed objects from incomplete ones. Such pending objects can be removed from heketi database using this command and later cleaned up on the nodes. The command has no intelligence and removes only the objects that have pending attribute set without cleaning references to the object in other object entries. This will be fixed in upcoming releases.
```

Comment:
* from db -> from the db
* have pending attribute set -> have the pending attribute set
* if heketi process -> if the heketi process
* in pending operations bucket -> in the pending operations bucket
* started, heketi process -> started, the heketi process
* from heketi database -> from the heketi database